pypeit/core/coadd.py

```diff
@@ -981,6 +1010,7 @@ def robust_median_ratio(flux, ivar, flux_ref, ivar_ref, mask=None, mask_ref=None
 
     return ratio
 
+# JFH This code should probably be deprecated since it is not used anywhere.
```

Comment:
No problems from me - happy to deprecate it!

Reply:
Removed.